### PR TITLE
Use RHN Pool name instead of Pool ID

### PR DESCRIPTION
**Pull request by elyezer**
Revert the changes so the ansible playbook uses the rhn_playbook
variable and update the job definition to proper pass that renamed
parameter.

### CODE REVIEW

**Comment by elyezer** [comment]
With these changes `git grep -i rhn_poolid` don't return anything

File: ci/ansible/roles/pulp/tasks/main.yaml [hunk 1]
```diff
@@ -29,18 +29,18 @@
     state: absent
   when: >
     ansible_distribution == "RedHat" and
-    rhn_poolid is defined and
+    rhn_pool is defined and
```
Comment (Ichimonji10) [inline ci/ansible/roles/pulp/tasks/main.yaml]: Ugh. I don't know how I missed this.

**Comment by Ichimonji10** [comment]
Merging.

**Comment by Ichimonji10** [comment]
Just to make sure: after applying this change, we'll pass in a regex like `'^SKU Name$'` when Jenkins provisions machines, right?

**Comment by elyezer** [comment]
> Just to make sure: after applying this change, we'll pass in a regex like '^SKU Name$' when Jenkins provisions machines, right?

Yes, the recent runs already had that in place and worked fine. Thank you for making sure everything is alright.

**Pull request merged**